packages/project-input-validator/.eslintignore

```diff
@@ -0,0 +1,2 @@
+test
```

Comment:
test should not be include here. 
Something like `test/fixtures` would be ok

Reply:
Excluded test folder